x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -5,7 +5,8 @@
  */
 
 import open from 'opn';
```

Comment:
This is an interesting one.

`puppeteer-core` is puppeteer without the chromium version bundled with it. Fantastic! However, I cannot seem to _map_ this module to the `@types/puppeteer` typedef since `@types/puppeteer-core` is waaaay behind. If any of y'all know how to do this I'd be forever grateful.